backend/os/location.go

```diff
@@ -19,7 +19,7 @@ type Location struct {
 	fileSystem vfs.FileSystem
 }
 
-// NewFile uses the properties of the calling location to generate a vfs.File (backed by an os.File). A string
+// NewFile uses the properties of the calling location to generate a vfs.File (backed by a osFile). A string
```

Comment:
why was this '.' removed?

Reply:
as importantly, why was n in 'an' removed? 
https://www.grammar.com/a-vs-an-when-to-use/

Reply:
Heh, I didn't even notice that.